### ISSUE TITLE
Lottie feature in csproj is missing if Fluent design is selected in the New Project Wizards

### ISSUE THREAD

**Issue by ArchieCoder** [unoplatform/uno.templates#683]
### Current behavior

The splash screen shows a warning that Lottie is missing.

![image](https://github.com/unoplatform/uno/assets/1608424/53eb3a3c-3098-47b0-80a4-b28bce578506)

### Expected behavior

No warning and progress ring visible

### How to reproduce it (as minimally and precisely as possible)

Create a new project and select Fluent design instead of Material design.

### Workaround

Add Lottie manually in the csproj.

### Works on UWP/WinUI

None

### Affected platforms

Android

**Comment by jeromelaban** [comment]
Yes, we're likely missing the Lottie dependency that only comes from material, but should always be added when extensions are select.

**Comment by dansiegel** [comment]
@ArchieCoder the Lottie package will only come in from the SDK if you have one of the 3 following UnoFeature flags:

- Lottie
- Material
- Cupertino

When you specifically choose to use the default Fluent design system there is no dependency that brings in the Lottie package as the Material and Cupertino packages both have transitive dependencies on the Lottie package. 

I am closing this as by design but please let us know if you feel there is still an issue and we can certainly reopen it and discuss it further.

**Comment by ArchieCoder** [comment]
If it is by design, there should be a documentation that the provided extended splash screen won't render correctly with the wizards.

It's not obvious for a new dev that they need to add Lottie for the ProgressRing to render.

**Comment by dansiegel** [comment]
@kazo0 can you please look into why the default Fluent Progress Ring isn't rendering here. This seems like it's most likely a bug in Toolkit

**Comment by jeromelaban** [comment]
It is likely that when toolkit is selected, we should be including lottie as well. We cannot have it as a transitive dependency from the toolkit because of platform constraints. The error message has been changed in https://github.com/unoplatform/uno/pull/16622 to make it more searchable.
